src/test/java/core/basesyntax/dao/FruitDaoImplTest.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.FruitsStorage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitDaoImplTest {
+    private FruitDao fruitDao = new FruitDaoImpl();
+
+    @After
+    public void afterEachTest() {
+        FruitsStorage.fruits.clear();
+    }
+
+    @Test
+    public void getFruitsStorageWithDifferentFruits_Ok() {
```

Comment:
```suggestion
    public void getStorageData_notEmpty_Ok() {
```
your test method names don't follow naming convention, change each name accordingly